### ISSUE TITLE
Using this with serverless-offline pattern includes are only copied once

### ISSUE THREAD

**Issue by CorruptedHeart** [floydspace/serverless-esbuild#277]
Given a package include like:
```
package:
  individually: true
  patterns:
    - "src/**/*.ejs"
```

The .ejs files will only be copied to output once, even if changes are made to the source files.

From the code it looks like it only copies if it doesn't already exist.

Was this for performance reasons?

Been thinking about how this could be fixed. Would the overhead of doing 2 stat calls per included file be worse than simply always copying? Doing the 2 stat calls would allow us to check that the file is newer before copying.

https://github.com/floydspace/serverless-esbuild/blob/master/src/index.ts#L324-L326

**Comment by samchungy** [comment]
Yeah I think it was because of performance reasons, in particular if users are using copying in `node_modules` which can contain a ridiculous ammount of files.

**Comment by samchungy** [comment]
> Yeah I think it was because of performance reasons, in particular if users are copying in `node_modules` which can contain a ridiculous ammount of files. Or if the user had big heavy artefacts
